components/ImageRC.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import Image from "next/image";
+
+// helper function used in the Image component
+function cn(...classes) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export default function ImageRC({

```

Comment:
Since we expect this  components folder to contain components for all sections of the project, it would be great to create a folder specific for events components. Eg `components/events`. 

I would also like to request you to change the name of the component. `ImageRC` can be misleading for a component that does this `Creating a reusable component for displaying events and descriptions.`

Reply:
I changed the name of the component to DisplayRC. The component will still use the event prop to determine whether to show an event or show a description.